package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "build": "lerna run build --stream",
     "build:demo": "lerna run build:demo --stream",
+    "build:demo_changed": "yarn build:demo --since HEAD^",
```

Comment:
Probably don't need this. I think the command reads better inline in the shell script.

Reply:
Makes sense. Removed.